Move to using Bleak BLE library

ukBaz opened the pull request:
I wanted to connect to the Aranet4 sensor from a Windows PC. 

This library seems to do a lot of the heavy lifting so I did an experiment to switch to using the [Bleak library](https://github.com/hbldh/bleak) (as it is cross platform) for reading the BLE data.

For my test I only modified the `aranet4\client.py` file and put a `test` function in there so I could run it with:

`python .\aranet4\client.py`

This PR should not be merged as I haven't converted all of the methods; just the ones I need for my experiments.

As I'd done this work I thought I would share it with you to see if there is any interest in switching from `bluepy` to `bleak` to make this library available for Linux, PC, and macOS.

Anrijs commented:
Merged this in separate branch for now. When all features will be implemented, I will move this to master.

ukBaz commented:
I've been doing some refactoring of the code which is breaking backwards compatibility and I don't know how important that is to you.

 - I've moved `aranet.py` inside the module and defined it as a console tool `aranetctl` so it is installed with pip installs from pypi.
 - `aranetctl` tool is installed on the path so can be used with simple call e.g. `aranetctl -h` to get help
 - Moved to using `argparse` in the `aranetctl` to get command line options rather than homebrew solution
 - Moved method names to be [pep8](https://www.python.org/dev/peps/pep-0008/) compliant
 - Put bleak as a dependency in `setup.py` so that it gets installed when pip installing aranet.
 - Move to using `dataclasses` rather than other data structures
 - Refactored the testing of "magic numbers"
 - Removed the `le16` as replaced functionality with Python `struct` module
 - Moved to using `IntEnum` for parameter types and status colors
 - 

I am doing this for my own benefit and so not sure how much of this you would want to take. Happy to have a feedback on what I'm doing so I know what changes you are OK with and what you are not.
It is still a work in progress but you can see what I have done so far at:
https://github.com/ukBaz/Aranet4-Python/tree/refactor

The other things on my plan is to 
- get the filtering options working for retrieving a subset of the logged data from the device
- replace `writeLE16` using the Python `struct.pack()` functionality
- write data to csv file
